data/afrique/homePage/index.json

```diff
@@ -480,52 +480,40 @@
         "visualProminence": "NORMAL",
         "radioSchedule": [
           {
-            "serviceId": "bbc_afrique_radio",
-            "publishedTimeStart": "2024-01-22T06:00:00.000Z",
-            "publishedTimeEnd": "2024-01-22T06:15:00.000Z",
-            "publishedTimeDuration": "PT15M",
-            "title": "Le Journal",
-            "brandPid": "p03p35rt",
-            "episodeTitle": "22/01/2024 GMT",
-            "episodePid": "w172z9v5y9h5lcf",
-            "broadcastPid": "p0h06c79",
-            "summary": "Le premier rendez-vous d'information de la matinée."
+            "id": "p086pmtv",
+            "state": "next",
+            "startTime": 1893488400000,
```

Comment:
By taking the latest update from the radio-schedule fabl-modules branch, this will convert the timestamp to an ISO string, but that's what we want - we need to make sure that the radio schedule will still render correctly